recipe_sha: 89d6a965f4a9b74d87364eade30096aa1b7c2a66a546356f7a99730acf50cf71  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
category: compiler error
log: |2-
    6:03:41 BIOCONDA INFO (OUT)   Error compiling Cython file:
      16:03:41 BIOCONDA INFO (OUT)   ------------------------------------------------------------
      16:03:41 BIOCONDA INFO (OUT)   ...
      16:03:41 BIOCONDA INFO (OUT)                                 if self.in_edge_count[i] == 0:
      16:03:41 BIOCONDA INFO (OUT)                                         merge_count += 1
      16:03:41 BIOCONDA INFO (OUT)                                         self.graph.remove_node( prestates[i] )
      16:03:41 BIOCONDA INFO (OUT) 
      16:03:41 BIOCONDA INFO (OUT)                                         if verbose:
      16:03:41 BIOCONDA INFO (OUT)                                                 print "Orphan state {} removed due to no edges \
      16:03:41 BIOCONDA INFO (OUT)               ^
      16:03:41 BIOCONDA INFO (OUT)   ------------------------------------------------------------
      16:03:41 BIOCONDA INFO (OUT) 
      16:03:41 BIOCONDA INFO (OUT)   yahmm/yahmm.pyx:2348:12: Syntax error in simple statement list
      16:03:41 BIOCONDA INFO (OUT)   Traceback (most recent call last):
      16:03:41 BIOCONDA INFO (OUT)     File "<string>", line 2, in <module>
      16:03:41 BIOCONDA INFO (OUT)     File "<pip-setuptools-caller>", line 34, in <module>
      16:03:41 BIOCONDA INFO (OUT)     File "/opt/conda/conda-bld/yahmm_1718121715058/work/setup.py", line 24, in <module>
        cmd_obj.run()
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/_distutils/command/build.py", line 132, in run
        self.run_command(cmd_name)
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 316, in run_command
        self.distribution.run_command(command)
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/dist.py", line 968, in run_command
        super().run_command(command)
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
        cmd_obj.run()
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/_distutils/command/build_ext.py", line 359, in run
        self.build_extensions()
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/_distutils/command/build_ext.py", line 479, in build_extensions
        self._build_extensions_serial()
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/setuptools/_distutils/command/build_ext.py", line 505, in _build_extensions_serial
        self.build_extension(ext)
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/Cython/Distutils/build_ext.py", line 130, in build_extension
        new_ext = cythonize(
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 1154, in cythonize
        cythonize_one(*args)
      File "/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 1321, in cythonize_one
        raise CompileError(None, pyx_file)
    Cython.Compiler.Errors.CompileError: yahmm/yahmm.pyx
    [1;31merror[0m: [1msubprocess-exited-with-error[0m

    [31m[0m [32mpython setup.py bdist_wheel[0m did not run successfully.
    [31m[0m exit code: [1;36m1[0m
    [31m>[0m See above for output.

    [1;35mnote[0m: This error originates from a subprocess, and is likely not a problem with pip.
    [1;35mfull command[0m: [34m/opt/conda/conda-bld/yahmm_1718121715058/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/python -u -c '[0m
  [34m  exec(compile('"'"''"'"''"'"'[0m
  [34m  # This is <pip-setuptools-caller> -- a caller that pip uses to run setup.py[0m
  [34m  #[0m
  [34m  # - It imports setuptools before invoking setup.py, to enable projects that directly[0m
  [34m  #   import from distutils.core to work with newer packaging standards.[0m
  [34m  # - It provides a clear error message when setuptools is not installed.[0m
  [34m  # - It sets sys.argv[0] to the underlying setup.py, when invoking setup.py so[0m
  [34m  #   setuptools doesn'"'"'t think the script is -c. This avoids the following warning:[0m
  [34m  #     manifest_maker: standard file '"'"'-c'"'"' not found".[0m
  [34m  # - It generates a shim setup.py, for handling setup.cfg-only projects.[0m
  [34m  import os, sys, tokenize[0m
  [34m  [0m
  [34m  try:[0m
  [34m      import setuptools[0m
  [34m  except ImportError as error:[0m
  [34m      print([0m
  [34m          "ERROR: Can not execute setup.py since setuptools is not available in "[0m
  [34m          "the build environment.",[0m
  [34m          file=sys.stderr,[0m
  [34m      )[0m
  [34m      sys.exit(1)[0m
  [34m  [0m
  [34m  __file__ = %r[0m
  [34m  sys.argv[0] = __file__[0m
  [34m  [0m
  [34m  if os.path.exists(__file__):[0m
  [34m      filename = __file__[0m
  [34m      with tokenize.open(__file__) as f:[0m
  [34m          setup_py_code = f.read()[0m
  [34m  else:[0m
  [34m      filename = "<auto-generated setuptools caller>"[0m
  [34m      setup_py_code = "from setuptools import setup; setup()"[0m
  [34m  [0m
  [34m  exec(compile(setup_py_code, filename, "exec"))[0m
  [34m  '"'"''"'"''"'"' % ('"'"'/opt/conda/conda-bld/yahmm_1718121715058/work/setup.py'"'"',), "<pip-setuptools-caller>", "exec"))' bdist_wheel -d /tmp/pip-wheel-72fnr0kj[0m
    [1;35mcwd[0m: /opt/conda/conda-bld/yahmm_1718121715058/work/
    Building wheel for yahmm (setup.py): finished with status 'error'
  [31m  ERROR: Failed building wheel for yahmm[0m[31m
  [0m  Running setup.py clean for yahmm
    Running command python setup.py clean
    'build/bdist.linux-x86_64' does not exist -- can't clean it
    'build/scripts-3.8' does not exist -- can't clean it
  Failed to build yahmm
  [31mERROR: Could not build wheels for yahmm, which is required to install pyproject.toml-based projects[0m[31m
  [0mException information:
  Traceback (most recent call last):
    File "$PREFIX/lib/python3.8/site-packages/pip/_internal/cli/base_command.py", line 180, in exc_logging_wrapper
      status = run_func(*args)
    File "$PREFIX/lib/python3.8/site-packages/pip/_internal/cli/req_command.py", line 245, in wrapper
      return func(self, options, args)
    File "$PREFIX/lib/python3.8/site-packages/pip/_internal/commands/install.py", line 429, in run
      raise InstallationError(
  pip._internal.exceptions.InstallationError: Could not build wheels for yahmm, which is required to install pyproject.toml-based projects
  Removed build tracker: '/tmp/pip-build-tracker-dk6gx091'
  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 590, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3638, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2506, in build
      utils.check_call_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 405, in check_call_env
      return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 381, in _func_defaulting_env_to_os_environ
      raise subprocess.CalledProcessError(proc.returncode, _args)
  subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/yahmm_1718121715058/work/conda_build.sh']' returned non-zero exit status 1.
# Last 100 lines of the build log.
